bindings/python/crocoddyl/core/diff-action-base.hpp

```diff
@@ -57,10 +57,22 @@ class DifferentialActionModelAbstract_wrap : public DifferentialActionModelAbstr
 
   boost::shared_ptr<DifferentialActionDataAbstract> default_createData() {
     return this->DifferentialActionModelAbstract::createData();
-  }
-};
+    void quasiStatic(const boost::shared_ptr<DifferentialActionDataAbstract>& data, Eigen::Ref<Eigen::VectorXd> u,
```

Comment:
You should bind it as virtual method, not pure virtual.
We want to run the default `quasiStatic` is it is not defined in any derived Python / c++ class.

(see the `createData` example). Note that you will need to bind `quasiStatic` in the derived c++ classes as well. This is due to the fact that in Python the base classes become `XXX_wrap`.